viewing_party/party.py

```diff
@@ -1,23 +1,211 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Input: parameters title, genre, and rating
+
+    Output: Returns a dictionary with the parameters as values, with
+    each key defined as a string of the same parameter name.
+    If a parameter is None/falsey, no dictionary is created.
+    '''
+    new_movie = {"title" : title, "genre" : genre, "rating" : rating}
+    if not all((title, genre, rating)):
+        new_movie = None
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    '''
+    Input: user_data (a dictionary with key "watched" and a list
+    as the value), and movie (a dictionary)
+
+    Output: Returns updated user_data dictionary, with the dictionary
+    movie appended to the list in the value of "watched".
+    '''
+    user_data = {
+        "watched": [movie]
+    }

```

Comment:
Close; this would be the case only if "watched" is not initialized yet. Otherwise, we'd want to append movie to `user_data["watched"]`.